chef/cookbooks/provisioner/templates/default/crowbar_join.suse.sh.erb

```diff
@@ -456,6 +456,18 @@ if [ "$MODE" == "setup" -o "$MODE" == "start" ]; then
     # Get our hostname
     HOSTNAME=$(hostname -f)
 
+    # stop ntpd before we run ntpdate, and start it again afterwards.
+<% if @target_platform_version.to_f < 12.0 then %>
+    ntp=ntp
+<% else %>
+    ntp=ntpd
```

Comment:
I'd probably call the variable `NTP_SERVICE`. Small update to change this would be welcome, but I give my +1 in advance since the rest is totally fine.